ui/src/main/webapp/css/style.css

```diff
@@ -371,3 +371,11 @@ quick-query {
     width: 500px;
 }
 
+.margin-top {
+    margin-top: 18px;
```

Comment:
margin-top and padding-bottom seem confusing as one is 10px and one is 18px. Could you update the names?